www/_posts/2024/2024-03-03-pyladies-at-pycon.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post.html
+title: "PyLadies at PyCon US 2024"
+tags: [PyLadies, Global, Community, Conference, PyCon, PyCon US]
+author: Naty Clementi
+author_link: https://twitter.com/ncclementi
+---
+
+# PyLadies at PyCon US 2024
+
+Are you heading to PyCon US 2024? 
+
+Here is a list of PyLadies that will be presenting their work at the conference. As 
+well as conference events that you won't want to miss. 
+
+Use this opportunity to learn more about the amazing work that PyLadies are 
+doing and don't hesitate to reach out. 
+
+
+## Tutorials
+
+### Wednesday 05/15
+
+**9am - 12:30pm**
+
+- [PyO3 101 - Writing Python modules in Rust](https://us.pycon.org/2024/schedule/presentation/113/) by [Cheuk Ting Ho](https://us.pycon.org/2024/speaker/profile/97/) - Room 319 
+
+
+**1:30pm - 5pm**
+
+- [Introduction to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more, from the comfort of your Python repl.](https://us.pycon.org/2024/schedule/presentation/55/) by [Naty Clementi](https://us.pycon.org/2024/speaker/profile/66/), [Gil Forsyth](https://us.pycon.org/2024/speaker/profile/64/) and [Phillip Cloud](https://us.pycon.org/2024/speaker/profile/65/) - Room 319
+
+- [Introduction to MicroPython: getting started with BBC micro:bit](https://us.pycon.org/2024/schedule/presentation/4/)  by [Juliana Karoline de Sousa](https://us.pycon.org/2024/speaker/profile/5/) - Room 310/311
+
+
+### Thursday 05/16
+
+**9am - 12:30pm**
+
+- [Monitoring carbon emissions: making the impact of your python code visible](https://us.pycon.org/2024/schedule/presentation/38/) by [Jessica Greene](https://us.pycon.org/2024/speaker/profile/44/) and [Chioma Onyekpere](https://us.pycon.org/2024/speaker/profile/45/) - Room 318
+
+
+
+## Talks and Charlas
+
+### Friday 05/17
+
```

Comment:
Maintainers summit - https://us.pycon.org/2024/events/maintainers-summit/ 

@InessaPawson  and myself and kara sowles